styles/globals.css

```diff
@@ -74,3 +74,13 @@ footer a {
 .d-none {
   display: none !important;
 }
```

Comment:
Can we avoid usage of `!important`?

Reply:
I will create a new PR to fix CSS `important` changes. Can't add to this PR.